scalyr_agent/monitor_utils/k8s.py

```diff
@@ -1309,11 +1328,14 @@ def query_api( self, path, pretty=0 ):
 
             global_log.log(scalyr_logging.DEBUG_LEVEL_3, "Invalid response from K8S API.\n\turl: %s\n\tstatus: %d\n\tresponse length: %d"
                 % ( url, response.status_code, len(response.text)), limit_once_per_x_secs=300, limit_key='k8s_api_query' )
```

Comment:
I wouldn't say we have any hard and fast rules here besides make sure we aren't consuming too many bytes in the customer's logs (after all, they do end up paying for the agent logs).  Since we do get new information every five minutes, I usually assume that we will be getting a pretty good picture of something like what URLs are causing this.  Typically when a customer has a problem, we probably have a day's worth of data to look at.

So, we end up not including the URL because it would consume more bytes and maybe give the chance of a large number of these entering the log (suppose there's a random string in the URL that is getting logged).  We rely on having lots of samples to paint the picture.

Reply:
Note, this is `DEBUG_LEVEL_3` logging which is almost never turned on anyway (and doesn't contribute to their logs, because it goes in a separate `agent_debug.log` which needs to be retrieved manually).

I'd say it's probably ok to remove the limit from this, and just log them all, unless we were worried about `agent_debug.log` having too much info in int.